pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -461,11 +462,11 @@ func (b *volumeBinder) BindPodVolumes(assumedPod *v1.Pod, podVolumes *PodVolumes
 }
 
 func getPodName(pod *v1.Pod) string {
-	return pod.Namespace + "/" + pod.Name
+	return framework.GetNamespacedName(pod.Namespace, pod.Name)
```

Comment:
It's better to just replace `getPodName` and `getPVCName` with `GetNamespacedName` directly.

Reply:
Makes sense.I'll do a quick replace